src/app.js

```diff
@@ -55,6 +57,19 @@ function createWindow() {
   WindowState.use('main', mainWindow)
 
   mainWindow.loadURL('https://mail.google.com')
+
+  mainWindow.webContents.on('dom-ready', function() {
+    fs.readFile(__dirname + '/../static/style.css', 'utf-8', function(
+      error,
+      data
+    ) {
+      if (!error) {
+        var formatedData = data.replace(/\s{2,10}/g, ' ').trim()
```

Comment:
@timche In #40 I updated to using Gulp for building the app.  This would be a good place for this logic to go as we could use gulp-cssmin to minify the css as part of the prebuild process.